src/scripts/components/MainDashboard.jsx

```diff
@@ -0,0 +1,193 @@
+/**
+ * @jsx React.DOM
+ */
+
+'use strict';
+
+var React = require('react/addons'),
+    Router = require('react-router'),
+	Button = React.createFactory(require('./Button.jsx')),
+	WidgetBoxes = React.createFactory(require('./WidgetBoxes.jsx')),
+	EditableModal = React.createFactory(require('./EditableModal.jsx')),
+	LogoutMessage = React.createFactory(require('./LogoutMessage.jsx'));
+
+
+// CSS
+require('./../../styles/main.less');
+require('./../../styles/variables.less');
+require('./../../styles/MainDashboard.less');
+require('./../../../node_modules/font-awesome/css/font-awesome.css');
+
+
+var MainDashboard = React.createClass({
+
+  //for themes, track state of the color clicked
+
+  getInitialState: function () {
+    return {
+	  currentColor: 'red',
+	};
+  },
+  
+  componentDidMount: function(){
+  
+    //for triggering open modals when we don't need the modal button (i.e. for logout)
+    this.refs.modal.visible('invisible');
+    this.refs.widgetOne.templateFunc(<WidgetOne></WidgetOne>);
+    this.refs.widgetTwo.templateFunc(<WidgetTwo></WidgetTwo>);
+    this.refs.widgetThree.templateFunc(<WidgetThree></WidgetThree>);
+  
```

Comment:
Seems like `this.refs.widgetOne`, `this.refs.widgetTwo` and `this.refs.widgetThree` has minimal differences. So could we **consolidate** those three? In the future, if we need to add another `WidgetBox`, are we going to add it manually here?